pkg/lbaas/acl/acl.go

```diff
@@ -147,6 +147,45 @@ func (a api) Create(ctx context.Context, definition Definition) (ACL, error) {
 	return payload, nil
 }
 
+func (a api) Update(ctx context.Context, identifier string, definition Definition) (ACL, error) {
+	endpoint, err := url.Parse(a.client.BaseURL())
+	if err != nil {
+		return ACL{}, fmt.Errorf("could not parse URL: %w", err)
+	}
+
+	endpoint.Path = utils.Join(path, identifier)
```

Comment:
this is probably broken everywhere in the old stuff, but client BaseURL could contain a Path in theory, which would be nuked with code like this. I trust your judgement on this, only wanted to make sure this is known.

Reply:
Haven' t really thought about this so far since BaseURL did not contain a path so far.
However it doesn't hurt anybody to make it more stable so I removed the path nuking from lbaas package